bin/jobsub_history

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/python3 -I
+
+#
+# jobsub_cmd -- generic wrapper for condor commands
+# COPYRIGHT 2022 FERMI NATIONAL ACCELERATOR LABORATORY
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+    jobsub generic wrapper for condor commands -- parse args like
+    -G group,  find schedd, call generic condor cmd
+"""
+# pylint: disable=wrong-import-position,wrong-import-order,import-error
+
+import argparse
+import os
+import sys
+import re
+import time
+
+PREFIX = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(os.path.join(PREFIX, "lib"))
+
+import fake_ifdh
+import get_parser
+
+
+def main() -> None:
+    """main line of code, proces args, etc."""
+    parser = get_parser.get_jobid_parser(add_condor_epilog=True)
+    parser.add_argument("-name", help="Set schedd name", default="")
```

Comment:
Should this be `--name`?

Reply:
Actually --name will get turned into -name later... I forget now what the case was that made us add that to the jobsub_q code, but I'm pretty sure we had some reason for it.

Reply:
Okay, there it is... https://github.com/marcmengel/jobsub_lite/commit/2701f479262a2f7184aa9c83ca6e2fc6f50465cd
so we were doing that so we detect the schedd even if they specify it as -name, because we're setting _condor_CREDD_HOST=schedd in the environment... So we probably do *not* need that in the jobsub_history code.